panel_handler.py

```diff
@@ -25,11 +25,13 @@ def start_panel(self):
 
     def during_measurement_panels(self):
         self.add((self.buttons, 60), (self.table, 170))
+        self.buttons.get_visible()[0].SetFocus()
 
     def after_panels(self):
         self.clear()
         self.add((self.buttons, 55), (self.table, 170))
         self.table.show_scrollbar()
+        self.buttons.get_visible()[0].SetFocus()

```

Comment:
buttony by si asi mali riesit tie metody ktore sa volaju v button_panel

Reply:
áno, ale konkrétne tu sú dva špecifické prípady v 2 stavoch aplikácie. 
kebyže sa to robí na inom mieste pribudne zbytočné podmienkovanie. 
tu na to stačí jeden príkaz, keďže je jasné kedy sa to zavolá. Môžeš to skúsiť presunúť, ale pribudnú ti riadky alebo premenná na kontrolu stavu ..

Reply:
ked sa ma zobrazit okno po merani metoda v handleri vola 
```
self.buttons.after_buttons()
self.panel_handler.after_panels()
```
takze tato metoda v button_panel sa vola az potom co sa nastavia buttony, takze nevidim problem preco by sa to nemohlo spravit uz v `self.buttons.after_buttons()`

Reply:
ok, mozeme aj tak. uz rozumiem.
prepisem 👍